general/general_ontology.ttl

```diff
@@ -0,0 +1,22 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix gax-core: <https://w3id.org/gaia-x/core#> .
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+
+general: a owl:Ontology ;
```

Comment:
please also add an entry in CODEOWNERS file to define the semantical expert for this "general" ontology. You can use me (Robert Schubert) as the technical reviewer.

Reply:
add Mirco AND Robert for general folder